janitor/finance.py

```diff
@@ -630,51 +630,52 @@ def inflate_currency(
     the currency's country.
 
     The provided country can be any economy name or code from the World Bank
-    list of economies:
-    https://databank.worldbank.org/data/download/site-content/CLASS.xls.
+    [List of economies](
+    https://databank.worldbank.org/data/download/site-content/CLASS.xls).
```

Comment:
@nvamsikrishna05 going to suggest a slightly different way of handling:

```suggestion
    Here is the [list of economies][econ]

    [econ]: https://databank.worldbank.org/data/download/site-content/CLASS.xls.
```